### PR TITLE
Gc log file

### PR DESCRIPTION
**Pull request by florian-zeidler**
@bjoernhaeuser @dfreudenberger @rebuy-de/it-platform This adds log rotate configuration for garbage collection logs.

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -4,7 +4,9 @@
   $solr_archive_file_name,
   $init_config,
   $solr_user,
-  $solr_port
+  $solr_port,
+  $gc_log_file_count = 5,
```
Comment (bjoernhaeuser) [inline manifests/config.pp]: that should not have defaults, because the defaults are already taken into account in the init.pp, right?

**Comment by svenwltr** [comment]
:+1:

**Comment by ghost** [comment]
:+1:

**Comment by bjoernhaeuser** [comment]
:+1:

**Pull request merged**